index.js

```diff
@@ -78,28 +83,29 @@ function decode (buffer) {
  * -62300 => 0xff0ca4
 */
 function encode (r, s) {
+  r = Buffer.concat([zeroBuffer1, r])
```

Comment:
Infact,  to avoid superfluous copies everywhere.
Can you just allocate this in the final `signature` buffer?
`6 + lenR + lenS` where `lenS = s.length + 1`